index.html

```diff
@@ -41,62 +45,86 @@
                 </div>
             </div>
         </div>
+        
+    </header>
+    <header id="desktop-header">
+        <div class="desktop-navigation flex-aic-jcsb" >
+            <a class="button" href="donate.html">Donate</a>
+            <a class="logo" href="index.html"><img src="./images/logo-h.png" alt="future for children logo-h" width="100"
+                    height="50"></a>
+            <a href="dreams.html">Dreams</a>
+            <a href="stories.html">Stories</a>
+            <a href="activities.html">Activities</a>
+            <a href="events.html">Events</a>
+            <a href="adoption.html">Adoption</a>
+            <a href="about.html">About Us</a>
+        </div>
     </header>
     <main>
         <section class="img-blocks img1">
```

Comment:
we have a problem with the fixed header. On my laptop, I can't see the head of the children so I think here we need to add an empty div with the height size of the header, I tried with margin and padding it doesn't work because of **_Parallax mode_** so I believe empty div will work.

Reply:
In fact, everyone does not necessarily have the same device as you...
Solution: 
You modify your mobile navigation bar
or
Change the second picture